ko/documentation/installation/index.md

```diff
@@ -118,12 +122,10 @@ $ sudo pacman -S ruby
 ### Homebrew (macOS)
 {: #homebrew}
 
-macOS 엘 캐피탄, 요세미티, 매버릭스에는 루비 2.0이 포함되어 있습니다.
-macOS 마운틴 라이언, 라이언, 스노 레퍼드는 루비 1.8.7을
-탑재하여 출시되었습니다.
+엘 캐피탄(10.11)부터 모하브(10.14)까지의 macOS 릴리스에서는 루비 2.0 이상이 포함되어 있습니다.
 
-많은 macOS 사용자들은 패키지 관리자로 [Homebrew][homebrew]를 사용합니다.
-Homebrew를 사용해 새로운 버전의 루비를 얻는 방법은 정말로 쉽습니다.
+[Homebrew][homebrew]는 macOS에서 일반적으로 사용되는 패키지 관리자입니다.
+Homebrew를 사용해 새로운 버전의 루비를 설치하는 것은 쉽습니다.
 
 {% highlight sh %}
 $ brew install ruby
```

Comment:
FreeBSD section is missing above the “Ruby on Solaris and OpenIndiana” section. Also Ruby on Solaris and OpenIndiana section should be updated: see [this commit](https://github.com/ruby/www.ruby-lang.org/commit/dc375e4aa909ce78d9cfb019a4a81940086830cf).

Reply:
06ee62b36b2da8f343 updated

Reply:
Can you also update `pkg install runtime/ruby-18` to `pkg install runtime/ruby`?
https://github.com/ruby/www.ruby-lang.org/blob/d3d96e59317231d25888f5de9e98adeb2315fddd/en/documentation/installation/index.md#ruby-on-solaris-and-openindiana

Reply:
753c66daf6e8724e7 Oops, I missed that. and also removed not exist sentence now ;)